kubernetes/helm/collabora-online/templates/deployment.yaml

```diff
@@ -34,6 +33,17 @@ spec:
         {{- include "collabora-online.selectorLabels" . | nindent 8 }}
         type: main
     spec:
+      {{- if .Values.deployment.hostAliases }}
+      hostAliases:
```

Comment:
Why so complianced, following should be the some with one line
```yaml
hostAliases: {{- toYaml .Values.deployment.hostAliases | nindent 6 }}
```

Reply:
Thanks

Reply:
Yaml does not need quoting -> and if golang / helm could parse it - golang / kubernetes will it parse it on the same way.

Reply:
I see , you mean quotes are not necessary ? what about ip then ? It should be list of ip and inside each element of ip list, there is list of hostnames. sorry If I didnot understand your point 
https://kubernetes.io/docs/tasks/network/customize-hosts-file-for-pods/

Reply:
ok thanks. I will update it after testing it